public-docs/swag-shop-03-custom-styling.md

```diff
@@ -0,0 +1,61 @@
+---
+id: swag-shop-3
+title: Custom Styling
+---
+
+# Adding custom styling
+
```

Comment:
Component styles can be overridden in two places — within the main theme override file, or within the specific component's file's `style` object. Styles are written in ES6 with [JSS](http://cssinjs.org/?v=v9.8.7). To learn how to write styles for components, refer to the [JSS documentation](http://cssinjs.org/json-api?v=v9.8.7).

Reply:
What we are specifically covering in this section is changing styling w/o editing components, so I don't think we want to tell people to edit the component